apps/web-giddh/src/app/shared/header/header.component.ts

```diff
@@ -291,7 +291,21 @@ export class HeaderComponent implements OnInit, AfterViewInit, OnDestroy, AfterV
         /* This will get the date range picker configurations */
         this.store.pipe(select(state => state.company.dateRangePickerConfig), takeUntil(this.destroyed$)).subscribe(config => {
             if (config) {
-                this.datePickerOptions = config;
+                // removed today and yesterday option from universal datepicker
+                // if (config && config.ranges && config.ranges.length && config.ranges[0] && config.ranges[0].name && config.ranges[0].name === 'Today') {
+                //         delete config.ranges[0];
+                // }
+
+                if (config && config.ranges && config.ranges[0] && config.ranges[0].name === 'Today' && config.ranges[1].name === 'Yesterday') {
```

Comment:
Add nullity check to `config.ranges[1]`

Reply:
config.ranges[0].name === ‘Today’  if this one get true then there will we atleat one or more elemet will be in array so for index 1 element will be there always if config.ranges[0].name === ‘Today’ will true